src/Plugin/Validation/Constraint/VideoProviderMatchConstraintValidator.php

```diff
@@ -20,13 +20,14 @@ class VideoProviderMatchConstraintValidator extends ConstraintValidator {
    */
   public function validate($value, Constraint $constraint) {
     $regexes = $constraint->getRegularExpressionsOption();
-    if (empty($value)) {
+    $url = $value->getUrl()->toString();
```

Comment:
Link fields are not the only field types that are supported in media_entity_embeddable_video. This code assumes it is.